docs/advanced-usage/compute-backends/slurm/index.md

```diff
@@ -33,3 +33,128 @@ Examples for CWL and Yadage can be found in
 
 > Please note that CERN Slurm cluster access is [not granted by
 default](https://batchdocs.web.cern.ch/linuxhpc/access.html)
+
+## Specifying environment
+
+There are several different ways to specify environments in the workflow
+specification for Slurm compute backend. As an alternative to the docker image,
+it is also possible to use Singularity container and to include the container
+file to `reana.yaml` inputs.
+
+```yaml hl_lines="6 10"
+
+  # Serial example
```

Comment:
If we show only Serial example here, whilst all examples below, it creates a certain disharmony.  We could either always cite four examples, but that would be kind of long... So what about the following structure:

- About (explaining what Slurm is about)
- Authentication (explaining that you have to have account and how to upload credentials)
- Specifying compute backend (just saying that we have `compute_backend` clause which needs to be set to `slurmcern` in job specifications, but not showing example; rather referring to the last section)
- Specifying job environment (saying that you can use singularity from CVMFS unpacked images, singularity that you upload yourself to the workspace, and docker by default that we convert to singularity for you);
- Specifying Slurm parameters (saying about partition and job time limits)
- Examples (where you show four examples for each workflow engine)